samples/bluetooth/unicast_audio_client/prj.conf

```diff
@@ -3,3 +3,4 @@ CONFIG_BT_DEBUG_LOG=y
 CONFIG_BT_CENTRAL=y
 CONFIG_BT_AUDIO=y
 CONFIG_BT_AUDIO_UNICAST_CLIENT=y
+CONFIG_BT_PACS=n # We don't publish any capabilities
```

Comment:
PACS is enabled by default, so we just disable it here for now. 

Will likely be changed in the future.